pkg/apiclient/info/info.proto

```diff
@@ -26,6 +26,7 @@ message GetUserInfoResponse {
     repeated string groups = 3;
     string email = 4;
     bool emailVerified = 5;
+    string serviceAccount = 6;
```

Comment:
I think it should probably be consistent with PodSpec which would be serviceAccountName:

https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/